README.md

```diff
@@ -4,10 +4,36 @@
 The Braket Algorithm Library provides Amazon Braket customers with pre-built implementations of prominent quantum algorithms and experimental workloads as ready-to-run example notebooks.
 
 ---
+### Braket algorithms
+
 Currently, Braket algorithms are tested on Linux, Windows, and Mac.
 
 Running notebooks locally requires additional dependencies located in [notebooks/textbook/requirements.txt](https://github.com/amazon-braket/amazon-braket-algorithm-library/blob/main/notebooks/textbook/requirements.txt). See notebooks/textbook/README.md for more information.
 
+| Algorithm | Notebook | References | 
```

Comment:
This table is a really nice improvement to this page - thanks for adding this!

One question - you've combined `advanced_algorithms` and `textbook` into the same list - would it be worth separating them into two lists to maintain the distinction?